parser_library/src/processing/preprocessors/endevor_preprocessor.cpp

```diff
@@ -18,55 +18,137 @@
 
 #include "preprocessor_options.h"
 #include "processing/preprocessor.h"
+#include "semantics/collector.h"
+#include "semantics/source_info_processor.h"
+#include "semantics/statement.h"
 
 namespace hlasm_plugin::parser_library::processing {
 
+namespace {
+struct stack_entry
+{
+    std::string name;
+    document doc;
+    document::iterator current;
+    stack_entry(std::string name, document doc_)
+        : name(std::move(name))
+        , doc(std::move(doc_))
+        , current(doc.begin())
+    {}
+
+    void next() { ++current; }
+    bool end() const { return current == doc.end(); }
+};
+
+std::string_view get_copy_member(const std::match_results<std::string_view::iterator>& matches)
+{
+    if (matches.size() != 4)
```

Comment:
does this still work when the line ends exactly after the member i.e. `-INC MEMBER<EOF>`?

Reply:
It works fine under Windows. The last match is just empty but it is still there. I'll update this when I find out how it looks like on Linux

Reply:
Seems fine on linux as well. 
The "include member" processing is now case insensitive for Endevor and DB2 statements as discussed.